Fatras/include/ActsFatras/Digitization/Channelizer.hpp

```diff
@@ -38,29 +38,39 @@ class Channelizer {
       const Hit& hit, const Acts::Surface& surface,
       const Acts::GeometryContext& gctx, const Acts::Vector3& driftDir,
       const Acts::BinUtility& segmentation, double thickness) const {
-    auto driftedSegment = m_surfaceDrift.toReadout(
+    // Drfted surface and scalor 2D to 3D segment
+    auto drifted = m_surfaceDrift.toReadout(
```

Comment:
Not strictly necessary, but spelling out the type might help in terms of clarity.